react/week4/github-userSearch/src/components/User.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import UserDetails from "./UserDetails";
+
+function User({ user }) {
+	const [userState, setUserState] = useState(false);

```

Comment:
This name is pretty confusing. Can we give it a more descriptive name?
What about?
```suggestion
	const [showUserDetails, setShowUserDetails] = useState(false);
```